src/frontend/src/lib/components/onramper/OnramperWidget.svelte

```diff
@@ -40,8 +40,8 @@
 			bitcoin: $btcAddressMainnet,
 			ethereum: $ethAddress,
 			erc20: $ethAddress,
-			icrc: $icrcAccountIdentifierText,
-			icp: $icrcAccountIdentifierText
+			icrc: $icpAccountIdentifierText,
```

Comment:
That cannot work. Old ICP account identity is only for ICP.

Reply:
yes, i mean, for now on onramper there is only ICP, but i see your point: i moved back the Principal for ICRC